packages/bpk-component-navigation-bar/readme.md

```diff
@@ -0,0 +1,55 @@
+# bpk-component-navigation-bar
+
+> Backpack navigation bar component.
+
+## Installation
+
+```sh
+npm install bpk-component-navigation-bar --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import { withButtonAlignment, withRtlSupport } from 'bpk-component-icon';
+import ArrowIcon from 'bpk-component-icon/sm/long-arrow-left';
+import { BpkButtonLink } from 'bpk-component-link';
+
+import BpkNavigationBar, { BpkNavigationIconButton } from 'bpk-component-navigation-bar';
+
+const AlignedArrowIcon = withButtonAlignment(withRtlSupport(ArrowIcon));
+
+export default () => (
+  <BpkNavigationBar
+    title="Backpack"
+    leadingButton={
+      <BpkNavigationIconButton onClick={() => {}}>
+        <AlignedArrowIcon />
+        <span className="visually-hidden">back</span>
+      </BpkNavigationIconButton>
+    }
+    trailingButton={
+      <BpkButtonLink onClick={() => {}}>Done</BpkButtonLink>
+    }
+  />
+)
+```
+
+## Props
+
+| Property              | PropType | Required | Default Value    |
+| --------------------- | -------- | -------- | ---------------- |
+| title                 | string   | true     | -                |
+| className             | string   | false    | null             |
+| leadingButton         | element  | false    | null             |
+| trailingButton        | element  | false    | null             |
+
+## BpkNavigationIconButton Props
```

Comment:
not sure if this is the right way to document this secondary component

Reply:
use a third level heading :) `### BpkNavigationBar` and `### BpkNavigationIconButton`